CodeFirst/Assement.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace TechnicalAssesment
+{
+    public enum GameResult
+    {
+        GameNotStarted,
+        DealerWon,
+        PlayerWon,
+        Tie,
+    }
+
+    public class ScoreBoard
+    {
+        public byte playerWins;
+        public int DealerWins;
+        public byte Ties { get; set; }
+    }
+
+    public class Deck
+    {
+        public List<int> cards;
+        public int MaxCards = 52;
+        public Deck()
+        {
+            cards = new List<int>();
+        }
+
+        public void Shuffle()
+        {
+            for (int i = 0; i < MaxCards; i++)
+            {
+                cards.Add(i);
+            }
+        }
+
+        public int DrawCard()
+        {
+            Random r = new Random();
```

Comment:
Your "Random" class can be a property created on construction of the "Desk" class. Creating "Random" class every card draw can be waste of resources